### ISSUE TITLE
Getting names for qantities and properties

### ISSUE THREAD

**Issue by GVladislavG** [xBimTeam/XbimEssentials#389]
Hello, xBim team!

I need to get quantitties and properties names for any IFC-class (any IPersistEntitty) in my code. 
All the quantitties and properties names like they are named in docs (for example IfcFooting https://standards.buildingsmart.org/IFC/RELEASE/IFC4_1/FINAL/HTML/schema/ifcstructuralelementsdomain/lexical/ifcfooting.htm)

But I have to do it without any IFC-file with filled properties and quantities. If I just create Entity of every class, properties and quantities are empty. Please tell and show is there any way to do this?

Thank you!

**Comment by martin1cerny** [comment]
Yes, there is an easy way to do this with xbim. All IFC properties and quantities are published in [several formats](https://standards.buildingsmart.org/IFC/RELEASE/IFC4/ADD2_TC1/HTML/link/listing-ifc4_add2.htm). And one of these formats is [IFC Property Templates](https://standards.buildingsmart.org/IFC/RELEASE/IFC4/ADD2_TC1/HTML/annex/annex-a/general-usage/IFC4_ADD2.ifc)! 

So, the easiest thing to do is to open this file with xbim and search for all [IfcPropertySetTemplates](https://standards.buildingsmart.org/IFC/RELEASE/IFC4/ADD2_TC1/HTML/link/ifcpropertysettemplate.htm). 

Caveat: It is used for both property sets and quantity sets.

**Comment by GVladislavG** [comment]
Thanks! 
And also similar question. How can I get tree of all IFC-classes (for IFC2x3 and IFC4)? 
In this case I need only names of the classes?
